<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VipWindow</class>
 <widget class="QMainWindow" name="VipWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VIP窗口</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="start">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>360</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>模拟</string>
    </property>
   </widget>
   <widget class="QPushButton" name="windowsView">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>400</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>查看窗口</string>
    </property>
   </widget>
   <widget class="QLabel" name="queueNum">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>320</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>280</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>前方排队人数：</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>210</x>
      <y>440</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dequeue">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>出队</string>
    </property>
   </widget>
   <widget class="QPushButton" name="enqueue">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>入队</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>160</y>
      <width>281</width>
      <height>71</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>银行排队模拟系统</string>
    </property>
   </widget>
   <widget class="QLabel" name="lineNum">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>290</y>
      <width>54</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>320</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>您的排队编号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="waitTime">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>260</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>250</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>预计等待时间：</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>23</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>VipWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>477</y>
    </hint>
    <hint type="destinationlabel">
     <x>415</x>
     <y>452</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
